lib/src/component/strictmode_component.dart

```diff
@@ -30,6 +32,17 @@ class StrictModeProps extends component_base.UiProps
   @override
   ReactComponentFactoryProxy componentFactory = react.StrictMode;
 
+  @override
+  PropsMetaCollection get staticMeta => throw UnimplementedError('StrictModeProps instances do not implement instance meta');
+
+  @override
+  void addUnconsumedProps(Map props, Iterable<PropsMeta> consumedProps) =>
+      throw UnimplementedError('StrictModeProps instances do not implement addUnconsumedProps');
+
+  @override
+  void addUnconsumedDomProps(Map props, Iterable<PropsMeta> consumedProps) =>
+      throw UnimplementedError('StrictModeProps instances do not implement addUnconsumedDomProps');
+
```

Comment:
```suggestion
```

Since this class inherits a concrete implementation that functions properly, there's no need for us to throw in overrides